README.md

```diff
@@ -1 +1,25 @@
-# score-backend
\ No newline at end of file
+# score-backend
+
+Tech stack:
+
+1. Typescript/Node.js
```

Comment:
wait u guys r using python lmao

Reply:
Oh oops I copied it from volume I think I forgot to change it